components/pipeline-service/production/kflux-ocp-p01/resources/update-tekton-config-pac.yaml

```diff
@@ -0,0 +1,9 @@
+---
+- op: add
+  path: /spec/platforms/openshift/pipelinesAsCode/settings
+  value:
+    application-name: Konflux OCP
+    custom-console-name: Konflux OCP
```

Comment:
@gbenhaim How should we name this one?

FYI:
p02 is "Konflux production internal"
rh01 is Red Hat Konflux"

Reply:
I think `Konflux OCP` is fine. It's the name of the app that appears in the github status check.